### PR TITLE
Parsing and loading Parkings realtime

### PR DESCRIPTION
**Pull request by nonifier**
This PR aims at load real-time parking data for Sytral. 

We keep the same mechanism as before with a separate go routine that fetches and parses a file that contains the information related to P+R parkings. This separate go routine can be configured aside from the one that refresh the departure with the following options:
```
      --parkings-refresh duration     time between refresh of parkings data (default 30s)
      --parkings-uri string           format: [scheme:][//[userinfo@]host][/]path
```  

This provides a new endpoint `/parkings/P+R` that returns all available parkings, as in :
```json
{
   "records" : [
      {
         "occupied_PRM" : 7,
         "occupied" : 118,
         "car_park_id" : "OULS",
         "available" : 181,
         "updated_time" : "2018-09-17T19:29:00+02:00",
         "available_PRM" : 0
      },
      {
         "updated_time" : "2018-09-17T19:29:00+02:00",
         "available_PRM" : 0,
         "available" : 361,
         "occupied_PRM" : 10,
         "occupied" : 99,
         "car_park_id" : "SOI"
      },
      {
         "occupied" : 77,
         "occupied_PRM" : 14,
         "available" : 499,
         "car_park_id" : "MEYP",
         "updated_time" : "2018-09-17T19:29:00+02:00",
         "available_PRM" : 0
      }
   ]
}
```
 
It's also possible to query parkings via their ids with : `/parkings/P+R?ids[]=CAR&ids[]=MEY`

Ultimately, this also adds the last time a refresh has been done for the parkings through `/status` :
```json
{
   "status" : "ok",
++"last_parking_update" : "2019-02-05T18:27:16.102271139+01:00",
   "last_departure_update" : "2019-02-05T18:27:16.423431818+01:00"
}

```

### CODE REVIEW

File: api.go [hunk 1]
```diff
@@ -18,8 +19,43 @@ type DeparturesResponse struct {
 
 // StatusResponse defines the object returned by the /status endpoint
 type StatusResponse struct {
-	Status         string    `json:"status,omitemty"`
-	LastDataUpdate time.Time `json:"last_data_update"`
+	Status              string    `json:"status,omitemty"`
```
Comment (xlqian) [inline api.go]: you probably mean omitempty
```suggestion
	Status              string    `json:"status,omitempty"`
```

Comment (nonifier) [reply api.go]: The eyes of the tiger ! :tiger:

File: api.go [hunk 2]
```diff
@@ -19,7 +19,7 @@ type DeparturesResponse struct {
 
 // StatusResponse defines the object returned by the /status endpoint
 type StatusResponse struct {
-	Status              string    `json:"status,omitemty"`
+	Status              string    `json:"status,omitempty"`
```
Comment (kinnou02) [inline api.go]: you didn't got an error from a linter for that?

Comment (nonifier) [reply api.go]: Nope....

File: api.go (hunk 1 above)
Comment (xlqian) [reply api.go]: I'm Dragon!

**Pull request merged**